bin/utils/cli-config-utils.js

```diff
@@ -8,9 +8,8 @@ const argv = require('getopts')(process.argv.slice(2));
 
 function mkConfigObj(opts) {
   if (!opts.client) {
-    const path = resolveDefaultKnexfilePath();
     throw new Error(
-      `No default configuration file '${path}' found and no commandline connection parameters passed`
+      `No configuration file found and no commandline connection parameters passed`
```

Comment:
This error message was rendering the config file name as `knexfile.undefined`.  This is because the `resolveDefaultKnexfilePath()` was expecting an argument for the desired extension.

However, Knex actually supports many different config file extensions, including `.ts` and `.coffee`.  So, it didn't really make sense to just show the `.js` extension.  Therefore, I removed the name from the error message.